ComSemApp/models.py

```diff
@@ -334,11 +334,54 @@ class SpeakingPracticeAttempt(models.Model):
     wpm = models.DecimalField(max_digits=5,decimal_places=2, validators=[MinValueValidator(0)], verbose_name='Words per Minute')
 
     def __str__(self):
-        return f"{self.id}: {self.expression} - {self.correct}% ({self.date.strftime('%d %b %Y')})"
+        return f"{self.pk}: {self.expression} - {self.correct}% ({self.date.strftime('%d %b %Y')})"
```

Comment:
why was this changed? jus curious

Reply:
While `id` _technically_ works, the proper way to do it django is with `pk`. This way brings in type hinting and some nice syntax highlighting.